packages/renderer/src/lib/image/image-utils.ts

```diff
@@ -172,6 +172,11 @@ export class ImageUtils {
     const icon = this.iconClass(imageInfo, context, viewContributions) || ImageIcon;
     const badges = this.computeBagdes(imageInfo, context, viewContributions);
 
+    // If the image isManifest = true, we will add a badge to the image that simply says "manifest"
+    if (imageInfo.isManifest) {
+      badges.push({ label: 'manifest', color: { light: 'bg-purple-600', dark: 'bg-purple-600' } });
```

Comment:
FYI unsure which colour we should use to indicate manifest, so just chose bg-purple-600 for now @deboer-tim @benoitf